app/components/AppBreadCrumb.tsx

```diff
@@ -29,7 +29,7 @@ const useCrumbs = (path: string): Crumb[] =>
     });
     if (parts.length > 1) {
       crumbs.push({
-        label: parts[1].split("?")[0],
+        label: decodeURI(parts[1].split("?")[0]),
```

Comment:
the need for decoding came up because the Auth0 user ID's all of the vertical bar `|` in them that was rendering as `%7C`